translations/ui-orders/en.json

```diff
@@ -373,6 +373,7 @@
   "errors.fundCannotBePaid": "One or more fund distributions on this order can not be encumbered, because there is not enough money in the budget",
   "errors.fundsNotFound": "The fund records are not found",
   "errors.idMismatch": "Mismatch between id in path and request body",
+  "errors.incorrectFundDistributionTotal": "Fund distribution total must add to 100% or totalPrice",
```

Comment:
Total price

Reply:
didn't get the comment

Reply:
since it's a message for user,  I suppose to use `Total price` instead of `totalPrice`

Reply:
it's a copy-paste from back-end and `totalPrice` is a reference to the field